src/primitives/lebesgue.jl

```diff
@@ -13,15 +13,19 @@ end
 
 Lebesgue(X) = Lebesgue{X}()
 
+Lebesgue(es::EuclideanSpace) = Lebesgue(EuclideanSpace{StaticInt{dimension(es)}})
+
```

Comment:
Let's add a `show` method for this, and maybe a method so `ℝ ^ (n::Int)` returns a `EuclideanSpace`

Reply:
Added in 
https://github.com/cscherrer/MeasureTheory.jl/pull/147/commits/a11d9a6d54b2fa891ed4c0df159c2e764b30f813

Reply:
Added in
https://github.com/cscherrer/MeasureTheory.jl/pull/147/commits/a11d9a6d54b2fa891ed4c0df159c2e764b30f813